common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -560,6 +560,16 @@ workflow_state = ? ` +
 
 	templateRangeCompleteReplicationTaskQuery = templateRangeCompleteTransferTaskQuery
 
+	templateGetTimerTaskQuery = `SELECT timer, timer_encoding ` +
```

Comment:
Okay that you added new query, but checkout pattern used for membership data in https://github.com/temporalio/temporal/blob/d6c4d670e9828f4f36584922fa0f1b1f76df1903/common/persistence/cassandra/cassandraClusterMetadata.go#L71 to have extensible queries.